admin_manual/installation/apps_management_installation.rst

```diff
@@ -47,32 +47,32 @@ If they are, then they will not be installed.
 Using Custom App Directories
 ----------------------------
 
-Use the **apps_paths** array in ``config.php`` to set any custom apps directory locations. 
-The key **path** defines the absolute file system path to the app folder. 
-The key **url** defines the HTTP web path to that folder, starting at the ownCloud web root. The key **writable** indicates if a user can install apps in that folder.
-
-.. note:: 
-   To ensure that the default **/apps/** folder only contains apps shipped with ownCloud, follow this example to setup an **/apps2/** folder which will be used to store all other apps.
-
-.. code-block:: php
-
-  <?php
-
-    "apps_paths" => array (
-        0 => array (
-                "path"     => OC::$SERVERROOT."/apps",
-                "url"      => "/apps",
-                "writable" => false,
-        ),
-        1 => array (
-                "path"     => OC::$SERVERROOT."/apps2",
-                "url"      => "/apps2",
-                "writable" => true,
-        ),
-    ),
+If you want to store apps in a custom directory, instead of ownCloud’s default (``/app``), you need to modify the ``apps_paths`` element in ``config/config.php``.
+There, you need to add a new associative array that contains three elements.
+These are:
+
+- ``path``: The absolute file system path to the custom app folder. 
+- ``url``: The request path to that folder relative to the ownCloud web root. 
+- ``writable``: Whether users can install apps in that folder. After the configuration is added, new apps will only install in a directory where ``writable`` is set to ``true``.
+
+The configuration example below shows how to add a second directory, called ``apps2``. 
+
+.. literalinclude:: ./examples/custom-app-directory-configuration.php
+   :language: php
+   :emphasize-lines: 9-13
+
+After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
+To do so, follow these steps:
+
+#. Enable maintenance mode
+#. Disable the apps that you want to move
```

Comment:
#. Create a new directory and assign the same rights and owncership as the apps folder to it 

Move the apps from the old to ``a new`` --> ``the new`` apps directory